HSLdevcom/jore-map-ui
# Fix rp save

Closes #591 
Closes #585 

* Fixed isDirty check, saveButton should be green/grey depending on if modifications has been made or not
   * done with lodash: _.isEqual()
   * Also there was a bug in markInvalidFields() method that I fixed
* I had a problem: inputFields needed to be refreshed when user presses "Muokkaa/Peruuta" button
   * solution: used force refresh (if inputContainer's disabled prop changed, validation was forced)

How to test:
Try to edit old and new routePaths, try to see if everything related to validating works
